packages/amplify-category-hosting/lib/S3AndCloudFront/helpers/configure-CloudFront.js

```diff
@@ -61,8 +65,19 @@ async function configure(context) {
       delete context.exeInfo.template.Outputs.CloudFrontDistributionID;
       delete context.exeInfo.template.Outputs.CloudFrontDomainName;
       delete context.exeInfo.template.Outputs.CloudFrontSecureURL;
-      const { AccessControl } = originalTemplate.Resources.S3Bucket.Properties;
+      delete context.exeInfo.template.Outputs.CloudFrontOriginAccessIdentity;
+
+      const {
+        AccessControl,
+        WebsiteConfiguration,
+      } = originalTemplate.Resources.S3Bucket.Properties;
       context.exeInfo.template.Resources.S3Bucket.Properties.AccessControl = AccessControl;
+      context.exeInfo.template.Resources.S3Bucket.Properties.WebsiteConfiguration =
+        WebsiteConfiguration;
+
+      context.print.warning('Static webhosting will be enabled for the hosting bucket once your remove CloudFront Distribution.');
+      context.print.info('Set it configuration:');
```

Comment:
What does this mean?

Reply:
Fixed typo, it should be "Set its configuration:".
Static webhosting is disabled when CloudFront is enabled, and when CloudFront is disabled, we need to enable back webhosting for the hosting bucket. 
When you enable static webhosting, you need to configure the index doc and error doc for it.